README.md

```diff
@@ -1,11 +1,26 @@
-<img src="https://github.com/fika-lang/assets/blob/master/logo.png?raw=true" width="150"/>
+<img src="https://github.com/fika-lang/assets/blob/master/logo.png?raw=true" width="150"/> &nbsp; ![](https://img.shields.io/github/forks/fika-lang/fika?style=social) ![](https://img.shields.io/github/stars/fika-lang/fika?style=social) ![](https://img.shields.io/github/watchers/fika-lang/fika?style=social) <br>
+
+![](https://img.shields.io/github/repo-size/fika-lang/fika) ![](https://img.shields.io/github/license/fika-lang/fika?color=red)<br>
+![](https://img.shields.io/github/issues/fika-lang/fika?color=green) ![](https://img.shields.io/github/issues-pr/fika-lang/fika?color=green) ![](https://img.shields.io/github/downloads/fika-lang/fika/total) ![](https://img.shields.io/github/last-commit/fika-lang/fika) ![](https://img.shields.io/github/contributors/fika-lang/fika)
+
```

Comment:
Nice!
Can we align this better?
Maybe keep the badges up top, followed by the logo and the github stats on the third line?

![Screenshot_2020-09-25 fika-lang fika](https://user-images.githubusercontent.com/5737888/94290877-6f7f6600-ff78-11ea-8791-241d83dba509.png)

Reply:
Ok, I'll look into it ..

.